plug.vim

```diff
@@ -986,6 +986,10 @@ function! s:update_impl(pull, force, args) abort
   let s:clone_opt = get(g:, 'plug_shallow', 1) ?
         \ '--depth 1' . (s:git_version_requirement(1, 7, 10) ? ' --no-single-branch' : '') : ''
 
+  if has('win32unix')
+    let s:clone_opt .= ' -c core.eol=lf -c core.autocrlf=input'
```

Comment:
Yes, I also thought that only `core.eol=lf` would suffice, but `core.autocrlf` actually overrides `core.eol`... and under Cygwin with `core.autocrlf=auto`, `crlf` is used by default. If you're curious, I can dig up the conversations on that topic I have found while doing that PR.